app/services/itunesApi.js

```diff
@@ -0,0 +1,6 @@
+import { generateApiClient } from '@utils/apiUtils';
+const repoApi = generateApiClient('itunes');
+
+export const getSongs = searchText => repoApi.get(`search?term=${searchText}`);
+
+export const getTrack = trackId => repoApi.get(`https://itunes.apple.com/lookup?id=909253`);
```

Comment:
the baseURL has already been set to `https://itunes.apple.com` so you can simply have something like 
```
itunesApi.get(`lookup?id=${trackId}`
```